samcli/commands/validate/validate.py

```diff
@@ -50,29 +55,36 @@ def do_cli(ctx, template):
     from .lib.exceptions import InvalidSamDocumentException
     from .lib.sam_template_validator import SamTemplateValidator
 
-    sam_template = _read_sam_file(template)
-
-    iam_client = boto3.client("iam")
-    validator = SamTemplateValidator(
-        sam_template, ManagedPolicyLoader(iam_client), profile=ctx.profile, region=ctx.region
-    )
-
-    try:
-        validator.is_valid()
-    except InvalidSamDocumentException as e:
-        click.secho("Template provided at '{}' was invalid SAM Template.".format(template), bg="red")
-        raise InvalidSamTemplateException(str(e)) from e
-    except NoRegionFound as no_region_found_e:
-        raise UserException(
-            "AWS Region was not found. Please configure your region through a profile or --region option",
-            wrapped_from=no_region_found_e.__class__.__name__,
-        ) from no_region_found_e
-    except NoCredentialsError as e:
-        raise UserException(
-            "AWS Credentials are required. Please configure your credentials.", wrapped_from=e.__class__.__name__
-        ) from e
-
-    click.secho("{} is a valid SAM Template".format(template), fg="green")
+    if lint:
+        _lint(ctx, template)
+    else:
+        sam_template = _read_sam_file(template)
+
+        iam_client = boto3.client("iam")
+        validator = SamTemplateValidator(
+            sam_template, ManagedPolicyLoader(iam_client), profile=ctx.profile, region=ctx.region
+        )
+
+        try:
+            validator.is_valid()
+        except InvalidSamDocumentException as e:
+            click.secho("Template provided at '{}' was invalid SAM Template.".format(template), bg="red")
+            raise InvalidSamTemplateException(str(e)) from e
+        except NoRegionFound as no_region_found_e:
+            raise UserException(
+                "AWS Region was not found. Please configure your region through a profile or --region option",
+                wrapped_from=no_region_found_e.__class__.__name__,
+            ) from no_region_found_e
+        except NoCredentialsError as e:
+            raise UserException(
+                "AWS Credentials are required. Please configure your credentials.", wrapped_from=e.__class__.__name__
+            ) from e
+
+        click.secho(
+            "{} can be transformed to a Cloudformation template. "
```

Comment:
could we still add "is a valid SAM Template" back and then continue to add the rest? There could be tools that are scrapping our output.